### PR TITLE
Fine-tune in R: end-to-end example

### PR DESCRIPTION
**Pull request by statist-bhfz**
This is tutorial mentioned in https://github.com/dmlc/mxnet/issues/5149

### CODE REVIEW

**Comment by matt32106** [comment]
@statist-bhfz out of sheer curiosity, how much accuracy do you get on your validation set?

**Comment by statist-bhfz** [comment]
@matt32106 It was around 86% on validation set and 83% on test set (public leaderboard) after 3 or 5 epochs. I'm still confused with R session crashes, so I can't perform more experiments.

**Comment by matt32106** [comment]
I had some trouble (R crashes) with the mean image because it needs to be the same size as data.shape, i don't know if this apply to your crashes!? (check issue #5214)


I did not try public LB yet but i get about 95% accuracy with bn-28, error rate:

http://pasteboard.co/FJjZNMJ8E.png


________________________________

**Comment by statist-bhfz** [comment]
No, I had another issue because I didn't subtract mean image.
